recipes/geocat-viz/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2022.04.0" %}
+
+package:
+  name: geocat-viz
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 38d44591dea798b88a9cb35799b02af295034be4e2a390f28c64782d01fa2b32
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: main # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/geocat-viz.git
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - cartopy
+    - cmaps
+    - matplotlib
+    - metpy
+    - numpy
+    - scikit-learn
+    - xarray
+
+test:
+  requires:
+    - pip
+  imports:
+    - geocat
+    - geocat.viz
+  commands:
+    - pip check
+  source_files:
+    - src
```

Comment:
```suggestion
```
I think these source files are not needed because you are no longer running the tests. If they are needed for the import tests to pass, then that is a problem because the end user will not have these files available during import.

Reply:
@carterbox , as one of the maintainers listed here, I can remove that. Is that okay?

Reply:
Yes, that's fine. It's common for PRs to have multiple editors.

Reply:
Done, thanks!